<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlankQtNGL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Simulate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Initialize</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="s_transformGB">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="4">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Recommended: 0.115</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Boundary Width</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Boundary Height</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y Offset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Viscosity Strength</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Particle Spacing</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Recommended: 0.15</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Boundary Depth</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Pressure Multiplier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Recommended: 1.5</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFluid_Sim">
    <property name="title">
     <string>Fluid Sim</string>
    </property>
   </widget>
   <addaction name="menuFluid_Sim"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
